src/runtime-rs/Makefile

```diff
@@ -178,7 +178,7 @@ ifneq (,$(DBCMD))
     CONFIGS += $(CONFIG_DB)
     # dragonball-specific options (all should be suffixed by "_DB")
     DEFMAXVCPUS_DB := 1
-    DEFBLOCKSTORAGEDRIVER_DB := virtio-blk
+    DEFBLOCKSTORAGEDRIVER_DB := virtio-mmio
```

Comment:
If we need to emphasize `mmio`, how about `virio-mmio-blk` as it is still a virtio-blk driver? The same comment applies to all the corresponding changes from `virtio-blk` to `virtio-mmio` in the PR.

Reply:
fixed